Hardcore.lua

```diff
@@ -1746,34 +1746,61 @@ function Hardcore_Frame_OnShow()
 		Hardcore_Level_Sort:Hide()
 		Hardcore_Zone_Sort:Hide()
 		Hardcore_TOD_Sort:Hide()
-		DeathListEntry3:Hide()
+		--DeathListEntry3:Hide()
```

Comment:
Why is this no longer needed?

Reply:
I copied the same behaviour as Rules button. And there that element isn't hidden.